rad-cli/src/bin/sync.rs

```diff
@@ -0,0 +1,53 @@
+extern crate rad_cli;
+
+use librad::profile::RadHome;
+
+use rad_clib::{keys::ssh::SshAuthSock};
+
+use rad_cli::{proc::some_or_exit, keys, id, tui, profile, project, seed};
+
+fn main() -> anyhow::Result<()> {
+    tui::headline("Syncing your local 🌱 projects");
+
+    let seed = "http://localhost:8778".to_string();
+    let home = RadHome::default();
+
+    let profile = some_or_exit(profile::default());
+    let storage = some_or_exit(keys::storage(&profile, SshAuthSock::default()));
```

Comment:
I have to say, I'm not a big fan of this pattern. Rust has the `?` syntax for this use-case. `Option` can also be converted to `Result` if needed, via `ok_or`.

Reply:
Ok. Agree. Found the right way to report through the CLI UI.